src/index.html

```diff
@@ -24,6 +24,41 @@ <h6 id="date"></h6>
       </ul>
     </header>
     <main>
+      <section class="main__section--forms main__sections">
```

Comment:
This looks like a good layout for the HTML of the form, I'm sure it would have looked amazing had we more time. None the less, the posts request concept was the thing we really wanted to get out of this and I think with the practice outside of this project we should be fine.